<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form Buku Tamu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        background-color: #f2f2f2;
        font-family: 'Segoe UI';
    }
    QPushButton {
        background-color: #4CAF50;
        color: white;
        padding: 6px;
        border-radius: 6px;
    }
    QPushButton:hover {
        background-color: #45a049;
    }
    QLineEdit, QDateEdit {
        background-color: white;
        border: 1px solid #ccc;
        border-radius: 4px;
        padding: 4px;
    }
    QLabel {
        font-size: 12pt;
        font-weight: bold;
    }
   </string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>571</width>
     <height>143</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="horizontalSpacing">
     <number>15</number>
    </property>
    <property name="verticalSpacing">
     <number>18</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="namaKaryawanLabel">
      <property name="text">
       <string>Nama Pengunjung</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="namaPengunjungLE"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="TanggalLaporanLabel">
      <property name="text">
       <string>Tanggal Kunjungan</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="TanggalKunjunganEdit">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="TanggalLaporanLabel_2">
      <property name="text">
       <string>Saran</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="SaranLE"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnsimpan">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnhapus">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnbersih">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Bersih</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>591</width>
     <height>172</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
